README.md

```diff
@@ -1,4 +1,78 @@
-# Python boilerplate for GitHub tasks
+# Washing Station
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+You own a car washing station. Washing cost calculation 
+takes a lot of time, and you decide to automate this
+calculation. The washing cost will depend on car comfort 
+class, car cleanness degree, wash station average rating
+and car wash distance from the center of the city.
+
+Create class `Car`, its constructor takes and stores
+3 arguments:
+1. `comfort_class` - comfort class of a car, from 1 to 7
+3. `clean_mark` - car cleanness mark, from very 
+dirty - 1 to absolute clean - 10
+4. `brand` - brand of the car
+
+Create class `CarWashStation`, its constructor takes and
+stores 4 arguments:
+1. `distance_from_cite_center` - how far station from
+the city center, from 1.0 to 10.0
+2. `clean_power` - `clean_mark` to which this car wash station
+washes
+3. `average_rating` - average rating of the station,
+from 1.0 to 5.0, rounded to 1 decimal
+4. `count_of_ratings` - number of people who rated
+
+`CarWashStation` should have such methods: 
+1. `serve_cars` Method that takes a list of `Car`'s, washes only
+cars with `clean_mark` < `clean_power` of wash station
+and returns total wash cost, rounded to 1 decimal
```

Comment:
and returns income of CarWashStation for serving this list of Car's, rounded to 1 decimal